docs/topics/whatsnew2020.md

```diff
@@ -27,7 +27,7 @@ See [Update to a new release](releases.md#update-to-a-new-kotlin-version) for de
 Kotlin 2.0.20 begins to introduce changes to improve consistency in data classes and replace the Experimental context 
 receivers feature.
 
-### Data class copy function to have the same visibility as constructor
+### Data class copy function will have the same visibility as constructor
```

Comment:
JFYI
it is `will have ...` in line 8, so I changed it here too

Reply:
For the reviewer: keep the original title.